src/elements/content-sidebar/activity-feed/annotation-thread/useAnnotationThread.js

```diff
@@ -98,6 +99,7 @@ const useAnnotationThread = ({
     };
 
     useAnnotatorEvents({
+        eventEmitter,
```

Comment:
I'm passing `eventEmitter` here in order for the current state of code to work, but I will create a seperate ticket for the refactoring of events handling as suggested by @jstoffan here: https://github.com/box/box-ui-elements/pull/3106#discussion_r1019439252